test/sdk-benchmarks/pom.xml

```diff
@@ -134,6 +134,16 @@
             <artifactId>netty-nio-client</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>dynamodb-enhanced</artifactId>
+            <version>${awsjavasdk.version}-PREVIEW</version>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-dynamodb</artifactId>
+            <version>1.11.748</version>
```

Comment:
Do we really need v1 here long term? Once we have a good baseline for v2, can we just track v2?

Reply:
We can remove it in the future, but while we're iterating on the performance we should have this in here.